index.html

```diff
@@ -1,20 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Ông Dev vui tính - Biết biển rộng được bao nhiêu nếu thuyền cứ mãi bám bờ, yo</title>
-    <link rel="stylesheet" type="text/css" href="css/index.css">
-</head>
-<body>
-<!--This layout is not 100% right, change it if you find a better way which is more make sense-->
+  <head>
+    <meta charset="UTF-8" />
+    <title>
+      Ông Dev vui tính - Biết biển rộng được bao nhiêu nếu thuyền cứ mãi bám bờ,
+      yo
+    </title>
+    <link
+      rel="shortcut icon"
+      href="assets/img/ong_dev.jpg"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" type="text/css" href="css/index.css" />
+  </head>
+  <body>
+    <!--This layout is not 100% right, change it if you find a better way which is more make sense-->
     <header>
-        <div id="menu">Menu</div>
-        <div id="cover">Cover</div>
+      <div id="menu">
+        Menu
+        <div class="menu">
```

Comment:
Duplicated divs: div with menu id and div with menu class.
I prefer using one and it should be id="menu"